server/runtime/src/main/server/bin/service.bat

```diff
@@ -0,0 +1,551 @@
+@echo off
+
+REM -------------------------------------------------------------------------
+REM  Red Hat Data Grid Service Script for Windows
+REM    It has to reside in %JDG_HOME%\bin
+REM    It is expecting that prunsrv.exe reside in one of:
+REM      %JDG_HOME%\bin
+REM      %JDG_HOME%\..\..\jbcs-jsvc-*\sbin
+REM     example command for installation:
+REM     service.bat install /jdguser admin /jdgpass admin 
+REM      /controller localhost:11222 /shutdown-file-name shutdown-command.txt 
+REM      /node-name doc /name JDG82 /desc "JDG 82" /loglevel DEBUG
+REM
+REM  v1	2021-06-11 initial edit
+REM
+REM Author: Lei Yu
+REM ========================================================
+setlocal EnableExtensions EnableDelayedExpansion
+
+set DEBUG=0
+if "%DEBUG%" == "1" (
+	echo "Debug info enabled"
+	echo on
+)
+
+set "DIRNAME=%~dp0%"
+if "%DEBUG%" == "1" (
+	echo DIRNAME "%DIRNAME%x"
+)
+
+set TEMP_JDG_HOME=%DIRNAME%..\
+
+pushd %TEMP_JDG_HOME%
+set "RESOLVED_JDG_HOME=%CD%"
+popd
+
+rem echo %RESOLVED_JDG_HOME%
+
+set DIRNAME=
+
+if "x%JDG_HOME%" == "x" (
+  set "JDG_HOME=%RESOLVED_JDG_HOME%"
+)
+rem echo %JDG_HOME%
+
+if "%DEBUG%" == "1" (
+	echo JDG_HOME="%JDG_HOME%"
+)
+
+rem echo "jdg home" %JDG_HOME%
+
+
+set PRUNSRV=
+rem Attempt to find prunsrv.exe under the same root directory that contains JDG_HOME.
+rem Note that only the last match will be used. This *typically* means the most recent version will be matched.
+rem Also note that only jbcs-jsvc-* will be used as a pattern, so jbcs-jsvc-1.0 and jbcs-jsvc-1.1 will match.
+for /d %%a in ( "%JDG_HOME%\..\jbcs-jsvc-*" ) do (
+  if "%DEBUG%" == "1" (
+    echo FOUND "%%~fa"
+  )
+  
+  if exist "%%~fa\sbin\prunsrv.exe" (
+    set "PRUNSRV=%%~fa\sbin\prunsrv.exe"
+  )
+)
+
+rem prunsrv.exe was not found above, so try to find it under JDG_HOME\bin.
+if "%PRUNSRV%" == "" (
+  if exist "%JDG_HOME%\bin\prunsrv.exe" (
+    set PRUNSRV="%JDG_HOME%\bin\prunsrv.exe"
+  ) else (
+    echo Please install native utilities into expected location %JDG_HOME%\..\jbcs-jsvc-1.1
+    goto cmdEnd
+  )
+)
+
+if "%DEBUG%" == "1" (
+	echo PRUNSRV %PRUNSRV%
+)
+
+echo(
+
+rem defaults
+set SHORTNAME=ISPN
+set DISPLAYNAME=ISPN
+rem NO quotes around the description here !
+set DESCRIPTION=Redhat data grid 8
```

Comment:
```suggestion
set DESCRIPTION=${infinispan.brand.name} ${infinispan.brand.version}
```

Reply:
As this is a default value for the description  of the window's service, I have changed it to 

set DESCRIPTION=Windows service for ISPN server

so that the value is not null if the user didn't enter a value via the parameters.